pkgs/development/python-modules/pyspnego/default.nix

```diff
@@ -10,18 +13,22 @@
 
 buildPythonPackage rec {
   pname = "pyspnego";
-  version = "0.1.6";
-  disabled = pythonOlder "3.6";
+  version = "0.2.0";
+
+  disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "jborean93";
     repo = pname;
     rev = "v${version}";
-    sha256 = "0pfh2x0539f0k2qi2pbjm64b2fqp64c63xxpinvg1yfaw915kgpb";
+    sha256 = "sha256-puv9aq53NbjSuN561XFou404N9pIxvvMjZMgnNx3SjM=";
   };
 
   propagatedBuildInputs = [
     cryptography
+    gssapi
+    krb5
```

Comment:
Is that right? `v0.2.0` added a dependency on the [`krb5` package in Pypi](https://github.com/jborean93/pyspnego/pull/14/files#diff-1a65d81b53483e58dc031cc4e50b94598ace0dcdf48b2aa7457aa685a1cf8edf), which is separate from the kerberos library that is used here.

The correct python dependency is this one, and it's not packaged yet:
https://github.com/jborean93/pykrb5

@fabaff: do you agree or am I missing something?